tasks/rh-sign-image/README.md

```diff
@@ -13,6 +13,11 @@ Task to create internalrequests to sign snapshot components
 | concurrentLimit | The maximum number of images to be processed at once                                      | Yes      | 4                    |
 | pipelineRunUid  | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       |                      |
 
+## Changes in 3.4.0
+* Added changes in order to eliminate the `translate-delivery-repo` script because the
+ `registry.redhat.io` and `acceess.redhat.io` repo are now available
```

Comment:
typo but isn't is access.redhat.com anyways?

Reply:
It's actually registry.access.redhat.com .

Reply:
Great catch. Fixed, thanks.